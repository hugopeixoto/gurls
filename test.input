http
there is no url here
http://www.google.com/
23:04 <dubaiss> Ler mais em: http://www.google.com/pt/
23:44 <Theorem> https://twitter.com/tqbf/status/619987355923910656?s=09 ddtc
