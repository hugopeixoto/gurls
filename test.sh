#!/usr/bin/env bash

diff test.expected <(./gurls < test.input)
