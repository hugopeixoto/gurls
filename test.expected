http://www.google.com/
http://www.google.com/pt/
https://twitter.com/tqbf/status/619987355923910656?s=09
http://pokemon.com
http://wikipedia.org
